_install/synology.md

```diff
@@ -14,24 +14,24 @@ This procedure if for Synology NAS running DiskStation Manager (DSM) version 6.X
 Open the DiskStation Manager interface. Use the Package Center to install the following packages (note they may already be installed):
 * Web Station
 * MariaDB 10
-* phpMyAdmin (only required for initial set-up of the database, may be removed once Webtrees is running).
+* phpMyAdmin (only required for initial set-up of the database, may be removed once webtrees is running).
 
 ## Webstation configuration
 
 1. In DSM, open "Web Station".
 2. Click on "General Setting" and check which PHP profile is being used ("Default PHP 7.4" in the below example):
 
-   ![Click on General Settings, check PHP profile.](../assets/img/synology/WS-GeneralSettings.png)
+   ![Click on General Settings, check PHP profile.](https://webtrees.net/assets/img/synology/WS-GeneralSettings.png)
```

Comment:
You only need to remove `..`.

You don't need to add `https://webtrees.net` - otherwise the site doesn't work for local/offline development.

Reply:
Thanks for the hint, I committed the change accordingly.